package.json

```diff
@@ -48,31 +46,20 @@
         "lint:packagejson": "pretty-quick --pattern package.json --pattern \"packages/*/package.json\" --pattern \"projects/*/package.json\" --pattern \"tools/*/package.json\" --pattern \"react/*/package.json\"",
         "lint:ts": "pretty-quick --pattern \"packages/**/*.ts\" && eslint -f pretty \"packages/**/*.ts\" && pretty-quick --pattern \"tools/**/*.ts\" && eslint -f pretty \"tools/**/*.ts\"",
         "lint:versions": "node ./scripts/lint-versions.js",
-        "new-package": "cd projects/templates && plop",
-        "postcustom-element-json": "node ./tasks/run-check-cem.js",
-        "postdocs:analyze": "node ./scripts/add-custom-properties.js --src=\"projects/documentation/custom-elements.json\"",
+        "new-package": "yarn workspace swc-templates plop",
         "postinstall": "husky || true && patch-package",
+        "postpack": "pinst --enable",
         "postpublish": "yarn prepublish:react && yarn publish:react && yarn postpublish:react",
         "postpublish:react": "git reset --hard HEAD^ && git prune && rimraf react",
-        "precustom-element-json": "node tasks/remove-custom-elements-json.js",
-        "preeleventy": "yarn docs:analyze",
         "prepack": "pinst --disable",
-        "postpack": "pinst --enable",
+        "prepublish:react": "yarn build:react && sed -i \"\" \"s/react/# react/g\" .gitignore && git commit -am \"chore: commit React watch\" --no-verify",
         "prepublishOnly": "rimraf react && yarn build && yarn custom-element-json && yarn verify-build-artifacts && yarn build:confirm",
-        "prepublish:react": "yarn build:react && sed -i \"\" \"s/react/# react/g\" .gitignore && git commit -am \"Commit React Wrappers\" --no-verify",
-        "prestorybook": "wireit",
-        "prestorybook:build": "cem analyze --outdir storybook/",
         "pretest:bench": "yarn build:tests && test -f test/benchmark/cli.js ||:",
-        "pretest:visual": "yarn build && yarn build",
-        "process-icons": "wireit",
-        "process-spectrum": "wireit",
         "publish:react": "yarn changeset publish --no-git-tag --tag snapshot --no-push",
         "push-to-remote": "git add . && git commit -m \"chore: release new versions #publish\" && git push",
```

Comment:
this command that was used in the original publish command includes a commit with `#publish` in it which is what the publish-docs-site workflow is listening for.

Reply:
You mentioned the "original" publish command. Does that mean we don't need this anymore?